ui/admin/app/components/form/worker/create-worker-led/index.hbs

```diff
@@ -209,9 +209,11 @@
 
     <Hds::Text::Body @tag='p' @size='300' class='p'>
       {{t 'resources.worker.form.steps.2.copy_registration_request'}}
-      <code>{{t
+      <Hds::Text::Code @size='300'>
```

Comment:
I notice there's some trailing space in the background that didn't look like it existed before:

After:
<img width="341" alt="image" src="https://github.com/user-attachments/assets/789c2946-96a7-45ab-9e08-37c0bb83adb1" />

Before:
<img width="341" alt="image" src="https://github.com/user-attachments/assets/d792faae-1987-4771-8a8c-b70973a2f249" />

Reply:
Weird. Taking a look.

Reply:
Fixed!
<img width="557" alt="Screenshot 2025-04-30 at 4 09 34 PM" src="https://github.com/user-attachments/assets/2375f20c-340c-4ed3-8d31-bda2f571a42c" />